templates/scenarios/js/sso-tab/src/components/sample/PersonCardGraphToolkit.jsx

```diff
@@ -7,23 +7,37 @@ export function PersonCardGraphToolkit(props) {
 
   return (
     <div className="section-margin">
-      <p>This example uses Graph Toolkit's&nbsp;
-        <a href="https://docs.microsoft.com/en-us/graph/toolkit/components/person-card" target="_blank" rel="noreferrer">person card component</a> with&nbsp;
-        <a href="https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/next/teamsfx/packages/providers/mgt-teamsfx-provider" target="_blank" rel="noreferrer">TeamsFx provider</a> to show person card.
+      <p>
+        This example uses Graph Toolkit's&nbsp;
+        <a
+          href="https://docs.microsoft.com/en-us/graph/toolkit/components/person-card"
+          target="_blank"
+          rel="noreferrer"
+        >
+          person card component
+        </a>{" "}
+        with&nbsp;
+        <a
+          href="https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/next/teamsfx/packages/providers/mgt-teamsfx-provider"
```

Comment:
The link is linked to `next/teamsfx` branch, should we use the latest main branch instead?

Reply:
Sure, thanks

Reply:
fixed